lib/email_checker.ex

```diff
@@ -1,23 +1,18 @@
 defmodule EmailChecker do
-  @validations [
-    &EmailChecker.Format.valid?/1,
-    &EmailChecker.MX.valid?/1,
-    &EmailChecker.SMTP.valid?/1,
-  ]
+
+  defp validations do
+    strategies = Application.get_env(:email_checker, :validations, [Format,MX,SMTP])
+    for v <- strategies, do: &Module.concat(EmailChecker,v).valid?/1
+  end
 
   def valid?(email) do
     expected_length =
       validations |> length
 
     case validations |> Stream.take_while(&( &1.(email) )) |> Enum.to_list do
-      list when length(list) == expected_length ->
-        true
-      _ ->
-        false
+      list when length(list) == expected_length -> true
```

Comment:
There is nothing about this where should be the values in a "case" in the syntax style guide.

I usually put them on a newline so we can easily see them without having to scroll horizontally.
Do you think it's better to have it at the end? If yes, why?